libplatsupport/plat_include/bcm2711/platsupport/plat/serial.h

```diff
@@ -18,25 +18,35 @@
  */
 #define UART_BUSADDR                0x7E215000
 
-#define UART_PADDR_0                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
+#define UART_PADDR_1                (UART_BUSADDR-BUS_ADDR_OFFSET+PADDDR_OFFSET)
 /*
  * BCM2711 TRM
  * The mini UART (UART1) is part of AUX
  * section 6.2.4. VideoCore interrupts:             29
  * section 6.3.   GIC-400 - VC peripheral IRQs:     96
  * => mini UART IRQ: 96 + 29 = 125
  */
-#define UART_IRQ_0                  (125)
+#define UART_IRQ_1                  (125)
 
 enum chardev_id {
     BCM2711_UART0,
+    BCM2711_UART1,
+    BCM2711_UART2,
+    BCM2711_UART3,
+    BCM2711_UART4,
+    BCM2711_UART5,
 
     NUM_CHARDEV,
     /* Aliases */
     PS_SERIAL0 = BCM2711_UART0,
+    PS_SERIAL1 = BCM2711_UART1,
+    PS_SERIAL2 = BCM2711_UART2,
+    PS_SERIAL3 = BCM2711_UART3,
+    PS_SERIAL4 = BCM2711_UART4,
+    PS_SERIAL5 = BCM2711_UART5,
     /* defaults */
-    PS_SERIAL_DEFAULT = BCM2711_UART0
```

Comment:
Can you add to the commit message that this is aligning the device numbering with the device tree and that `BCM2711_UART0` was previously referring to the serial1 device.

Reply:
Commit message contains the stated information now.